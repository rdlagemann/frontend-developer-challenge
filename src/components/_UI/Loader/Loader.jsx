import React from 'react'
import loaderIcon from './loading-process.svg'

export default function Loader(props) {
  return <img src={loaderIcon} {...props} alt="" />
}
