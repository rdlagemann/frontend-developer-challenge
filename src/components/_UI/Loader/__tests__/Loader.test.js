import React from 'react'
import { create } from 'react-test-renderer'
import Loader from '../Loader'

describe('Loader', () => {
  it('renders', () => {
    const output = create(<Loader />)
    expect(output).toMatchSnapshot()
  })
})
