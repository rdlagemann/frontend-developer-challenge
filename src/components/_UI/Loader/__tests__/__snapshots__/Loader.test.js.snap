// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders 1`] = `
<img
  alt=""
  src="loading-process.svg"
/>
`;
